Samples/BashTestSpecificToDotNet2x/test.json

```diff
@@ -1,12 +0,0 @@
-{
-  "name": "BashTestSpecificToDotNet2x",
-  "enabled": true,
-  "requiresSdk": true,
-  "version": "2.x",
```

Comment:
Should we maybe change the version string and keep this sample around? The purpose is to test that skipping bash tests specific to a .NET version (irrespective of whether it's EOL or not) works correctly.